core/trino-main/src/main/java/io/trino/spiller/FileSingleStreamSpillerFactory.java

```diff
@@ -116,10 +115,10 @@ public FileSingleStreamSpillerFactory(
                 createDirectories(path);
             }
             catch (IOException e) {
-                throw new IllegalArgumentException(format("could not create spill path %s; adjust %s config property or filesystem permissions", path, SPILLER_SPILL_PATH), e);
+                throw new IllegalArgumentException(format("could not create spill path %s; adjust spiller-spill-path config property or filesystem permissions", path), e);
             }
             if (!isAccessible(path)) {
-                throw new IllegalArgumentException(format("spill path %s is not accessible, it must be +rwx; adjust %s config property or filesystem permissions", path, SPILLER_SPILL_PATH));
+                throw new IllegalArgumentException(format("spill path %s is not accessible, it must be +rwx; adjust spiller-spill-path config property or filesystem permissions", path));
```

Comment:
Using `SPILLER_SPILL_PATH` constant in error messages looks fine to me. When we rename the property, we may forget to update here.